insights/parsers/rhn_entitlement_cert_xml.py

```diff
@@ -1,9 +1,30 @@
 import xml.etree.ElementTree as ET
-from .. import Parser, parser, LegacyItemAccess
+from .. import XMLParser, parser, LegacyItemAccess
+
+
+def _parse_content(content):
```

Comment:
As the example of the base class I gave above, this method should be moved into the class (as a class method "self._parse_dom") and can be renamed to "_parser_dom". And just return the `rhn_cert`